mdx-models/src/main/java/com/mx/accessors/account/TransactionBaseAccessor.java

```diff
@@ -98,6 +98,17 @@ public AccessorResponse<MdxList<Transaction>> list(String accountId, Transaction
     throw new AccessorMethodNotImplementedException();
   }
 
+  /**
+   * Pending transactions
```

Comment:
I would change the description to `Get all pending transactions for an account` or something similar.

Reply:
Added

Reply:
Sorry, I was talking about the Javadoc description which currently reads `Pending transactions`. I think the API description can be `List pending transactions`. See `recent()` as an example.